test/e2e/test_cluster_configs/network/cilium.json

```diff
@@ -1,59 +0,0 @@
-{
```

Comment:
This test doesn't make sense: there's only one cilium config, and that's for implementing NetworkPolicy (plus cilium IPAM). There is no exclusive networkPlugin=cilium cluster configuration.